addons/arsenal/functions/fnc_addListBoxItem.sqf

```diff
@@ -6,7 +6,7 @@
  *
  * Arguments:
  * 0: Config category, must be "CfgWeapons", "CfgVehicles", "CfgMagazines", "CfgVoice" or "CfgUnitInsignia" <STRING>
- * 1: Classname <STRING>
+ * 1: Classname (must be in config case) <STRING>
```

Comment:
Eh, just do the casing instead of requiring it to be cased already.

Reply:
Close to the entirety of the arsenal uses config case (I know one exception are the favorites, but I don't think much or anything else). There is no reason to convert to lower case, especially if it's for a simple cache, it's not worth the performance loss.

Reply:
I didn't say anything about lowercasing, only config-casing.

Reply:
Still not worth it.

Reply:
The cost is the same whether it's done before the fnc or within the fnc, the latter being a much better UX.